cards/multilang-product-prominentvideo/component.js

```diff
@@ -12,21 +12,17 @@ class multilang_product_prominentvideoCardComponent extends BaseCard['multilang-
    * @param profile profile of the entity in the card
    */
   dataForRender(profile) {    
-    profile.featuredMessage = {
-      description: 'subtitle'
-    }
-    
     const videoUrl = profile.videos && profile.videos[0]?.video?.url;
     const youtubeVideoId = videoUrl?.split('watch?v=')[1];
     const youtubeUrl = youtubeVideoId ? 'https://www.youtube.com/embed/' + youtubeVideoId : null;
```

Comment:
What do you think about making a formatter for this? Something like `getYouTubeUrl` or something like that? It would take in `profile.videos` and spit out the `youtubeUrl`.

Reply:
checking with product!

Reply:
added a formatter